app/pastes/pasteRepository.js

```diff
@@ -16,6 +17,22 @@ class PasteRepository {
     }
 
     async findAll({ ids, name, authorId }) {
+        if (await this.cacheProvider.exists(`${ids}_${name}_${authorId}`)) {
```

Comment:
`if (await...)` is not fancy at all.
Move it into `const isPasteCached = await this.cache...` or smth like it.